sdk/core/azure/CHANGELOG.md

```diff
@@ -507,7 +505,7 @@ version before installing the latest release.
 * Azure Storage has moved.
     https://github.com/Azure/azure-storage-python
 
-    http://azure-storage.readthedocs.org/en/latest/
+    https://azure.github.io/azure-sdk-for-python/ref/Storage.html
```

Comment:
I don't see the point of changing a link from a Changelog written in 2015

Reply:
removed the link